sentry-android-core/src/test/java/io/sentry/android/core/AndroidCpuCollectorTest.kt

```diff
@@ -24,7 +24,7 @@ class AndroidCpuCollectorTest {
             whenever(mockBuildInfoProvider.sdkInfoVersion).thenReturn(Build.VERSION_CODES.LOLLIPOP)
         }
         fun getSut(buildInfoProvider: BuildInfoProvider = mockBuildInfoProvider) =
```

Comment:
```suggestion
        fun getSut() =
```
and remove the `mockBuildInfoProvider` variable above